src/pages/all.jsx

```diff
@@ -40,7 +40,7 @@ const AllPage = () => {
                         <h2>Tell Your Story</h2>
                         <p>&quot;Jenkins Is The Way&quot; is a global showcase of how developers and engineers are building, deploying, and automating great stuff with Jenkins.</p>
                         <p>Share your story and we'll send you a free Jenkins Is The Way T-shirt.</p>
-                        <p>Submit a pull request about your project's goals and technical challenges, and the unique solutions you encountered with Jenkins.</p>
+                        <p><Link to={{ pathname: 'https://stories.jenkins.io/admin/#/collections/user-story' }} target="_blank">Share the story of your project's goals, technical challenges, and the unique solutions you encountered with Jenkins.</Link></p>
```

Comment:
Link is only used for inside the gastby app (it does some fancy react magic so it doesn't need to download everything again).
```suggestion
                        <p><a href="/admin/#/collections/user-story" target="_blank">Share the story of your project's goals, technical challenges, and the unique solutions you encountered with Jenkins.</a></p>
```

I'm not a fan of target=_blank, I think users should be allowed to decide when a new page is opened and when its not. When its forced, you can't unforce it.

Reply:
Agree, thanks very much @halkeye!